src/varint.rs

```diff
@@ -0,0 +1,49 @@
+use varint_rs::VarintReader;
```

Comment:
Are we happy with `warint_rs`? I wonder why `postcard` ships its own.

Reply:
We have robust unit tests around varints, and `varint_rs` seems to pass them all. I have no concerns with it personally.